src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNav.razor.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+
+namespace Bit.Client.Web.BlazorUI
+{
+    public partial class BitNav
+    {
+        [Parameter] public string? SelectedKey { get; set; }
+        [Parameter] public string? AriaLabel { get; set; }
+        [Parameter] public bool IsOnTop { get; set; }
+        [Parameter] public List<NavLink> NavLinks { get; set; } = new List<NavLink>();
+        [Parameter] public EventCallback<NavLink> OnClick { get; set; }
+        [Parameter] public RenderFragment<NavLink>? HeaderTemplate { get; set; }
+        protected override string RootElementClass => "bit-nav";
+
+        protected virtual async Task Toggle(NavLink navLink)
+        {
+            if (IsEnabled is false) return;
+
+            if (navLink.Disabled) return;
+
+            if (navLink.Links?.Any() ?? false)
```

Comment:
always use { }

Reply:
Can I ask why?

Reply:
no :)

Reply:
just search about it and accept it as a convention in this project :)